presenting-tosw.adoc

```diff
@@ -63,13 +64,14 @@ The pathway of this guidebook therefore is:
 1. *Attract users* to your software because it solves the problems they have.
 2. *Guide participants* who care about open source and sharing good solutions so that they can be effective enthusiasts for your software.
 3. *Grow contributors* from this rich, fertile user and enthusiast base, by making sure when any of those folks look into how the software is made, they see themselves represented and can imagine how they could fit in, too.
+^^^Are these points sequential? If not, this should be a bulleted list. -BKP
```

Comment:
As a comment for the record, I think we do see it as sequential; not as a requirement but more of an effect of the focus. First focus on users, then make sure participants are enabled, so that your lowered barriers are ready when contributors start to filter through this funnel.

If numbers seem to "these are your orders", in the next iteration we can change them to bullets with descriptive words that show the first, then, finally.

Reply:
I'll try some connective words to show the flow a bit. "Conjunction Junction, what's your function?"